src/bot.js

```diff
@@ -15,7 +15,8 @@ class SieroClient extends AkairoClient {
 
         // Set up command handler
         this.commandHandler = new CommandHandler(this, {
-            directory: './commands/',
+            directory: './build/dist/commands/',
+            loadFilter: filepath => filepath.slice(-3) === '.js',
```

Comment:
What does this do?

Reply:
Akairo by default will load every single file regardless of type from the directory and does stuff with them. Since in `tsconfig.json`, `declaration` is on (generates typescript type files automatically), the folder includes `.ts` files. When it tries to load them, it breaks because Node doesn't understand ts.

Reply:
oic, cool